utils.go

```diff
@@ -456,3 +457,18 @@ func TimeElapsed(now, then time.Time, abbreviationFormat string, fullTime bool)
 	}
 	return parts[0] + text
 }
+
+func voteVersion(params *chaincfg.Params) uint32 {
```

Comment:
If callers don't currently care about the current vote version, we might be able to do without this function.

Reply:
the caller doesn't require it, but it's still relevant in fetching all agendas, it's used here https://github.com/planetdecred/dcrlibwallet/pull/216/files#diff-51a9ecb87bb262c791b92cb917a6eca6d3f685e3bfce2638ddbce6c74fd115aeR206

we would need that method to be exported upstream, so we only have to maintain one copy of it